src/modal/modal-window.ts

```diff
@@ -82,7 +83,9 @@ export class NgbModalWindow implements OnInit,
     } else {
       elementToFocus = body;
     }
-    elementToFocus.focus();
+    setTimeout(() => {
```

Comment:
What is the purpose of the `setTimeout` here? It seems to make all the unit tests fail